### ISSUE TITLE
cannot use publish command

### ISSUE THREAD

**Issue by HarukiSato0804** [tableau/tabcmd#147]
I was facing the following error when I tried to publish my twb file to tableau online.
I can publish the same twb file with tabcmd 20211.0.0.0 with the same command.

```bash
tabcmd publish Book1.twb  -o  -r "xxxxx"  --parent-project-path "xxxxx" --db-username "xxxx" --db-password "xxxxx"
```

```bash
	403132: Forbidden
		Book1 failed to establish a connection to your datasource.
Traceback (most recent call last):
  File "~/.pyenv/versions/3.9.9/lib/python3.9/site-packages/src/execution/tabcmd_controller.py", line 44, in run
    namespace.func.run_command(namespace)
  File "~/.pyenv/versions/3.9.9/lib/python3.9/site-packages/src/commands/datasources_and_workbooks/publish_command.py", line 60, in run_command
    new_workbook = server.workbooks.publish(new_workbook, args.filename, publish_mode)
  File "~/.pyenv/versions/3.9.9/lib/python3.9/site-packages/tableauserverclient/server/endpoint/endpoint.py", line 190, in wrapper
    return func(self, *args, **kwargs)
  File "~/.pyenv/versions/3.9.9/lib/python3.9/site-packages/tableauserverclient/server/endpoint/endpoint.py", line 232, in wrapper
    return func(self, *args, **kwargs)
  File "~/.pyenv/versions/3.9.9/lib/python3.9/site-packages/tableauserverclient/server/endpoint/endpoint.py", line 232, in wrapper
    return func(self, *args, **kwargs)
  File "~/.pyenv/versions/3.9.9/lib/python3.9/site-packages/tableauserverclient/server/endpoint/workbooks_endpoint.py", line 449, in publish
    server_response = self.post_request(url, xml_request, content_type)
  File "~/.pyenv/versions/3.9.9/lib/python3.9/site-packages/tableauserverclient/server/endpoint/endpoint.py", line 147, in post_request
    return self._make_request(
  File "~/.pyenv/versions/3.9.9/lib/python3.9/site-packages/tableauserverclient/server/endpoint/endpoint.py", line 69, in _make_request
    self._check_status(server_response)
  File "~/.pyenv/versions/3.9.9/lib/python3.9/site-packages/tableauserverclient/server/endpoint/endpoint.py", line 85, in _check_status
    raise ServerResponseError.from_response(server_response.content, self.parent_srv.namespace)
tableauserverclient.server.endpoint.exceptions.ServerResponseError: 

	403132: Forbidden
		Book1 failed to establish a connection to your datasource.
```

This twb connects to redshift.
I am using tabcmd installed with the following command.

```bash
pip install  git+https://github.com/tableau/tabcmd.git@d493cddfc662593b050f998068c94833d48f110e   
```

My tabcmd version is 
```zsh
❯ tabcmd -v                                                                                                                                        
Tableau Server Command Line Utility v2.0.6.dev9
```

**Comment by jacalata** [comment]
This is because we aren't properly passing in the username/password with the db, will get it fixed soon.

**Comment by jacalata** [comment]
This is fixed in the latest release (2.0.7, and probably 2.0.6 too)- I'll close this now, but feel free to re-open it if there's anything missing or further work needed.
